### DISCUSSION TITLE
Problem with  FacePosition in mt mobile device (Android)

### DISCUSSION THREAD

**Discussion by igorfr33** [vladmandic/human#351] [General]
I'm struggling for resolve a problem for Gesture in Human to make the facin in mobile devices, the direction i look the screen show the other direction, and i switch the logical in IF insinde the archive human.esl in those lines
![image](https://user-images.githubusercontent.com/81191666/230206766-37b0e749-5a5f-4fcf-bafe-b6f28ea62433.png)

**Comment by vladmandic** [comment]
thanks - i've made some adjustments, it will be part of the next release.
